I think we should keep `govet`:

```
golangci-lint run --max-issues-per-linter=0 --max-same-issues=0
validator/client/runner.go:93:4: lostcancel: this return statement may be reached without using the cancel var defined on line 101 (govet)
                        return // Exit if context is canceled.
                        ^
validator/client/runner.go:101:4: lostcancel: the cancel function is not used on all paths (possible context leak) (govet)
                        slotCtx, cancel := context.WithDeadline(ctx, deadline)
                        ^
validator/client/runner.go:134:16: lostcancel: the cancel function returned by context.WithDeadline should be called, not discarded, to avoid a context leak (govet)
                                domainCtx, _ := context.WithDeadline(ctx, deadline)
                                           ^
validator/client/runner.go:148:14: lostcancel: the cancel function returned by context.WithDeadline should be called, not discarded, to avoid a context leak (govet)
                        rolesCtx, _ := context.WithDeadline(ctx, deadline)
                                  ^
4 issues:
* govet: 4
```

And `unused` as well (excluding the `third_party` directory).

--- a/.golangci.yml
+++ b/.golangci.yml
@@ -6,9 +6,9 @@ linters:
     - errcheck
     - ineffassign
     - govet
-    - unused
   disable:
     - staticcheck
+    - unused
   exclusions:
     generated: lax
     presets:

--- a/.golangci.yml
+++ b/.golangci.yml
@@ -3,13 +3,12 @@ run:
   go: 1.23.5
 linters:
   enable:
-    - errcheck    # Passes - checks for unchecked errors
-    - ineffassign # Passes - detects unused assignments
+    - errcheck
+    - ineffassign
+    - govet
+    - unused
   disable:
-    # Disable the default linters that currently fail
-    - govet       # 9 issues - vet examinations like unreachable code, context leaks
-    - staticcheck # 50 issues - various static analysis checks
-    - unused      # 26 issues - unused constants, variables, functions
+    - staticcheck
   exclusions:
     generated: lax
     presets:

--- a/validator/client/runner.go
+++ b/validator/client/runner.go
@@ -90,6 +90,7 @@ func (r *runner) run(ctx context.Context) {
 		select {
 		case <-ctx.Done():
 			log.Info("Context canceled, stopping validator")
+			//nolint:govet
 			return // Exit if context is canceled.
 		case slot := <-v.NextSlot():
 			if !r.healthMonitor.IsHealthy() {
@@ -98,7 +99,7 @@ func (r *runner) run(ctx context.Context) {
 			}
 
 			deadline := v.SlotDeadline(slot)
-			slotCtx, cancel := context.WithDeadline(ctx, deadline)
+			slotCtx, cancel := context.WithDeadline(ctx, deadline) //nolint:govet
 
 			var span trace.Span
 			slotCtx, span = prysmTrace.StartSpan(slotCtx, "validator.processSlot")
@@ -131,7 +132,7 @@ func (r *runner) run(ctx context.Context) {
 
 			// Start fetching domain data for the next epoch.
 			if slots.IsEpochEnd(slot) {
-				domainCtx, _ := context.WithDeadline(ctx, deadline)
+				domainCtx, _ := context.WithDeadline(ctx, deadline) //nolint:govet
 				go v.UpdateDomainDataCaches(domainCtx, slot+1)
 			}
 
@@ -145,7 +146,7 @@ func (r *runner) run(ctx context.Context) {
 				continue
 			}
 			// performRoles calls span.End()
-			rolesCtx, _ := context.WithDeadline(ctx, deadline)
+			rolesCtx, _ := context.WithDeadline(ctx, deadline) //nolint:govet
 			performRoles(rolesCtx, allRoles, v, slot, &wg, span)
 		case e := <-v.EventsChan():
 			v.ProcessEvent(ctx, e)

